src/main/java/seedu/address/model/pdf/Deadline.java

```diff
@@ -36,8 +39,9 @@ public Deadline() {
      */
     public Deadline(String jsonFormat) {
         this.date = LocalDate.parse(jsonFormat.split(Deadline.PROPERTY_SEPARATOR_PREFIX)[Deadline.PROPERTY_DATE_INDEX]);
-        this.isMet = Boolean.parseBoolean(jsonFormat
-                .split(Deadline.PROPERTY_SEPARATOR_PREFIX)[Deadline.PROPERTY_ISMET_INDEX]);
+        this.status = STATUS_REMOVE;
```

Comment:
So the format for `this.status` should follow the code that you removed. i.e. The same way to obtain the value as `this.isMet`